carbon-design-system/carbon-addons-iot-react
# fix(icondropdown): fixed outline and dropshadow and top direction

Closes #1771

**Summary**

- Fixed dropshadow from appearing and disappearing with the footer. The focus outline has also been restored when opening the menu. Furthermore the 'top' direction would break with multiple lines of text, also fixed.

**Change List (commits, features, bugs, etc)**

- See above

**Acceptance Test (how to verify the PR)**

- Head on over to https://deploy-preview-1785--carbon-addons-iot-react.netlify.app?path=/story/watson-iot-experimental-icondropdown--default 
- Open menu, observe drop shadow exists. Also observe blue outline exists. Mouse over items, footer should show as well with the outline and dropshadow updates. Change orientation to 'top' from 'bottom'. Repeat steps.
